test/lit.cfg

```diff
@@ -1681,6 +1681,7 @@ elif run_os == 'linux-androideabi' or run_os == 'linux-android':
         '-target', config.variant_triple,
         '-sdk', shell_quote(config.variant_sdk)])
     config.target_swift_ide_test = ' '.join([
+        'env', 'SDKROOT={}'.format(shell_quote(config.variant_sdk)),
```

Comment:
I noticed that the two failing stdlib C++ Interop tests don't pass the Android sysroot anywhere so added this, but haven't tested this change since the prebuilt snapshot toolchain doesn't come with `swift-ide-test`.

Reply:
We already do this for sil-opt, so doing this for `swift-ide-test` should be safe.